sqlalchemy_utils/view.py

```diff
@@ -149,7 +152,11 @@ def create_indexes(target, connection, **kw):
         for idx in table.indexes:
             idx.create(connection)
 
-    sa.event.listen(metadata, 'before_drop', DropView(name))
+    sa.event.listen(
```

Comment:
This change isn't functional and is only because CI was failing on the lint job for too many characters in one row.